pkg/agent/datamodel/types.go

```diff
@@ -823,7 +832,7 @@ func (p *Properties) GetClusterID() string {
 		} else if len(p.AgentPoolProfiles) > 0 {
 			h.Write([]byte(p.AgentPoolProfiles[0].Name))
 		}
-		r := rand.New(rand.NewSource(int64(h.Sum64())))
+		r := rand.New(rand.NewSource(int64(h.Sum64()))) //nolint:gosec // I think we want rand not crypto/rand here
```

Comment:
in theory I think we wanted to ban math/rand to avoid confusion but think we'd then need a wrapper which is out of scope

this isn't security sensitive either way

Reply:
also newline for comment please :)